packages/plugin-rows/src/editor/controls/extended-settings.tsx

```diff
@@ -29,9 +29,11 @@ const Content = styled.div<{ name: string }>(({ name, ...props }) => {
     backgroundColor: theme.backgroundColor,
     position: 'relative',
     padding: '15px',
-    width: '98%',
+    width: '90%',
     maxWidth: '600px',
-    margin: 'auto'
+    margin: 'auto',
+    height: '90%',
```

Comment:
I'm not so sure about this, looks weird when there are no settings (apart from the clone / delete controls)... maybe `maxHeight` instead?.

Reply:
ups, of course maxHeight